cosmos/ingestion/ingest/ingest.py

```diff
@@ -35,7 +35,7 @@
 logging.getLogger("ingest.process.detection.src.torch_model.train.data_layer.xml_loader").setLevel(logging.ERROR)
 
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.ERROR)
+logger.setLevel(logging.DEBUG)
```

Comment:
Does this make the logging overly verbose?

Reply:
Thanks @iross - a sloppy mistake on my part. I've changed it back to DEBUG.